go.mod

```diff
@@ -24,7 +24,8 @@ require (
 	go.uber.org/thriftrw v1.25.0
 	go.uber.org/yarpc v1.55.0
 	go.uber.org/zap v1.13.0
-	golang.org/x/net v0.7.0
+	golang.org/x/net v0.19.0
+	golang.org/x/oauth2 v0.15.0
```

Comment:
@Groxx do these version upgrades look fine to you? would cause issues with monorepo?

Reply:
I think these are breaking

Reply:
@mantas-sidlauskas I think due to the google cloud archiver, we're going to have a hard time getting past changes here:
```
// not sure we can currently upgrade past this:
cloud.google.com/go/iam v0.3.0
```

Reply:
@mantas-sidlauskas  I think upgrading this bumps the IAM library which breaks some other thing in our dep tree, at the moment I don't know we can upgrade to this min version: 
```
go get golang.org/x/oauth2@v0.15.0
Setting GOPROXY to https://goproxy.uberinternal.com
go: downloading golang.org/x/oauth2 v0.15.0
go: upgraded cloud.google.com/go v0.104.0 => v0.110.2
go: upgraded cloud.google.com/go/compute v1.7.0 => v1.20.1
go: upgraded cloud.google.com/go/iam v0.3.0 => v0.13.0 // breaks, we can't currently get past 0.3.0
...
```